// Copyright 2009 Google Inc. All Rights Reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// Protocol buffer used to represent minimize dns results.

syntax = "proto2";

import "pagespeed/core/pagespeed_output.proto";

package pagespeed;

message MinimizeResourcesDetails {
  extend ResultDetails {
    // In order to attach it to Result.details
    optional MinimizeResourcesDetails message_set_extension = 5073262;  // int('Min')
  }

  // Number of hosts that contain violations.
  optional int32 num_violation_hosts = 1;

  // List of wasteful url requests
  repeated string violation_urls = 2;
}
